agent/lib/kontena/workers/service_pod_manager.rb

```diff
@@ -31,7 +31,8 @@ def start
 
       subscribe('service_pod:update', :on_update_notify)
       subscribe('service_pod:event', :on_pod_event)
-      every(30) do
```

Comment:
Maybe it's safe to revert back to `every`.  There is still exclusive block that should block if something weird happens with task scheduling.